wdl/transforms/draft2/src/main/scala/wdl/transforms/draft2/wdlom2wom/WdlDraft2WomCommandTaskDefinitionMaker.scala

```diff
@@ -27,12 +28,19 @@ object WdlDraft2WomCommandTaskDefinitionMaker extends WomCommandTaskDefinitionMa
     // Figure out the start line of the workflow in the source file
     val t: Option[Terminal] = AstTools.findTerminals(wdlTask.ast).headOption
 
+    def stringifyMetaValues(meta: Map[String, String]): Map[String, MetaValueElement] = {
+      meta map {
+        case (key, value) =>
+          key -> MetaValueElement.MetaValueElementString(value)
+      }
+    }
+
     CallableTaskDefinition(
       name = wdlTask.fullyQualifiedName,
       commandTemplateBuilder = Function.const(wdlTask.commandTemplate.validNel),
       runtimeAttributes = wdlTask.runtimeAttributes.toWomRuntimeAttributes(wdlTask),
-      meta = wdlTask.meta,
-      parameterMeta = wdlTask.parameterMeta,
+      meta = stringifyMetaValues(wdlTask.meta),
```

Comment:
Actually, come to think of it, shouldn't this throw an error instead of trying to make a composite type into a string?

Reply:
But the input type is `Map[String, String]`, how does the method know if this is supposed to be a composite type?

Reply:
I think you're right - the draft 2 parser classes seem to always fill in `wdlTask.meta` with a `Map[String, String]` so it seems appropriate for the "to wom" process to always convert that String into a `MetaStringElement`